web/src/api/hfiCalculatorAPI.ts

```diff
@@ -44,12 +46,20 @@ const url = '/hfi-calc/daily'
 
 export async function getDailies(
   startTime: number,
-  endTime: number
+  endTime: number,
+  stationCodes: number[]
 ): Promise<StationDaily[]> {
   const { data } = await axios.get<StationDailyResponse>(url, {
     params: {
       start_time_stamp: startTime,
-      end_time_stamp: endTime
+      end_time_stamp: endTime,
+      station_codes: stationCodes
+    },
+    // have to add a paramsSerializer to axios get request and stringify (using querystring library) the params in order
+    // for the stationCodes array to be formatted correctly, per
+    // https://stackoverflow.com/a/51444749
```

Comment:
We can do this as a one-liner without having to add another dependency: https://stackoverflow.com/a/63402277

Reply:
tbh I prefer it the way it is - the code is cleaner

Reply:
I'm less concerned about how it looks vs. adding another dependency. We're currently running into updating `react-scripts` because we introduced `csv-string`, which depends on Node streams. The more packages we add the more pain we experience with maintenance. Let's keep this for now and dev chat as a team.